ignition/model/lifecycle.py

```diff
@@ -2,16 +2,19 @@
 STATUS_COMPLETE = 'COMPLETE'
 STATUS_FAILED = 'FAILED'
 STATUS_UNKNOWN = 'UNKNOWN'
+LIFECYCLE_MESSAGE_VERSION = "1.0.0"
 
 class LifecycleExecuteResponse():
 
-    def __init__(self, request_id, associated_topology=None):
+    def __init__(self, request_id, associated_topology=None, version=None):
```

Comment:
Should this be `version=LIFECYCLE_MESSAGE_VERSION` and then below have `self.version = version`?

Reply:
Addressed in commit 4.